shared/settings/passphrase/container.js

```diff
@@ -4,11 +4,12 @@ import UpdatePassphrase from '.'
 import {compose, lifecycle, connect, type RouteProps} from '../../util/container'
 import HiddenString from '../../util/hidden-string'
 
-type OwnProps = RouteProps<{}, {}>
+type OwnProps = RouteProps<{heading: string}, {}>
 
-const mapStateToProps = state => ({
+const mapStateToProps = (state, {routeProps}: OwnProps) => ({
```

Comment:
🔥 the `: OwnProps` here, flow should know from the type param in `connect` below